README.md

```diff
@@ -1,2 +1,40 @@
 # mAWS.jl
```

Comment:
Did you consider the more explicit name `MockAWS.jl`? This way people would know what this package does just based on the name, without having to read the description.

Reply:
Cool package by the way!

Reply:
Just changed it :)